Chapters/Counter/Exo-Counter.pillar

```diff
@@ -61,80 +51,82 @@ Object subclass: #Counter
 
-Select menu item 'accept' to store this class comment in the class.
+Figure *@CounterClassCommented* shows the class with its comment.
 
-Figure *@CounterClassCommented* shows the class with its comment. 
 +Counter class has now a comment! Well done.>file://figures/CounterClassCommented.png|label=CounterClassCommented+
 
-!!! Define protocols and methods
-In this part you will use the browser to learn how to add protocols and methods.
+!!!Defining protocols and methods
+
+In this part you will use the Browser to learn how to add protocols and methods.
+
+The class we have defined has one instance variable named ==count==, and we're going to use that variable to keep count. We'll increment it, decrement it, and show its current value. But in Pharo we need to remember three things:
 
-The class we have defined has one instance variable named ==count==. You should remember that in Pharo, (1) everything is an object, (2) that instance variables are private to the object, and (3) that the only way to interact with an object is by sending messages to it.
+# ''Everything'' is an object
+# Instance variables are ''completely private'' to the object
+# The ''only'' way to interact with an object is by ''sending messages'' to it
 
-Therefore, there is no other mechanism to access the instance variable values from outside an object than sending a message to the object. What you can do is to define messages that return the value of the instance variable. Such methods are called ''accessors'', and it is a common practice to always define and use them. We start to create an accessor method for our instance variable ==count==.
+And so there is no other mechanism to access our instance variable from outside of our counter than by sending a message to the object. What must do is define a method that returns the value of the instance variable. Such methods are called ''accessors''. So, let's define an accessor method for our instance variable ==count==.
 
-A method is usually sorted into a protocol. These protocols are just a group of methods without any language semantics, but convey important navigation information for the reader of your class. Although protocols can have any name, Pharo programmers follow certain conventions for naming these protocols. If you define a method and are not sure what protocol it should be in, first go through existing code and try to find a fitting name.
+A method is usually placed into a ''protocol''. These protocols are just a group of methods - they have no meaning in Pharo, but they do convey important information to the readers of your class. Although protocols can have any name, Pharo programmers follow certain conventions when naming protocols. If you define a method and are not sure what protocol it should be in, first take a look through existing code and see if you can find an appropriate protocol that already exists.
 
-!!! Create a method
+!!!!Create a method
```

Comment:
I prefer to have this as a large entry (even if it is logically nested in the other). 
Because this way we have a point to jump.